docs/demos/usb-camera-demo-rpi4.md

```diff
@@ -40,6 +40,13 @@ The following will be covered in this demo:
     ```sh
     kubectl taint nodes --all node-role.kubernetes.io/master-
     ```
+     > **Note** If this command does not work you may need to setup kubenetes config and environment variables using the commands below	
```

Comment:
Can we move these above the `kubectl taint` as these first three lines are always required and are requested actions at the end of kubeadm init?

Reply:
Moved to the init step.